src/workerd/io/actor-sqlite.c++

```diff
@@ -687,7 +686,7 @@ ActorCacheInterface::DeleteAllResults ActorSqlite::deleteAll(WriteOptions option
   // Reset alarm state, if necessary.  If no alarm is set, OK to just leave metadata table
   // uninitialized.
   if (localAlarmState != kj::none) {
-    metadata.setAlarm(localAlarmState);
+    metadata.setAlarm(localAlarmState, options.allowUnconfirmed);
```

Comment:
I guess in the current state of the code, allowUnconfirmed would always be false here, since we assert against it earlier in deleteAll().  But I assume it would eventually be meaningful when allowUnconfirmed is implemented for deleteAll().

Reply:
Yes, I debated whether to make this outright `false`, but figured that if we want to support `allowUnconfirmed` with `deleteAll` at some point, this is the correct code to have.